AlphaWallet/Settings/Types/Config.swift

```diff
@@ -36,7 +36,7 @@ struct Config {
     }
 
     static var locale: Locale {
```

Comment:
I know you mentioned this in #https://github.com/AlphaWallet/alpha-wallet-ios/issues/2655#issuecomment-803947279, but I don't understand.

When do we use `Config.locale` and when do we use `Config.getLocale()`? In other words, why do we want `Config.locale` to return `Locale.current` when the user has overridden the language in in-app Settings > Advanced > Change Language?